composer.json

```diff
@@ -10,12 +10,11 @@
 	"scripts": {
 		"cs:fix": "php-cs-fixer fix",
 		"cs:check": "php-cs-fixer fix --dry-run --diff",
-		"lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './composer/*' -print0 | xargs -0 -n1 php -l",
```

Comment:
@juliushaertl is this not enough? The linter still traverses the composer dir

Reply:
https://github.com/nextcloud/viewer/blob/master/.php-cs-fixer.dist.php#L15 is actually the place of the cs fixer config, so it would need to also exclude `composer/` there